library/board.py

```diff
@@ -0,0 +1,182 @@
+import typing
+
+import blessed
+import numpy as np
+
+
+class Board:
+    """Represents the board that the game of tic-tac-toe is played on."""
+
+    contents: np.array = np.full((9, 9), ".")  # type: ignore
+    player1_turn: bool
+    game_over: bool
+
+    def __init__(self) -> None:
+        self.contents = np.full((9, 9), ".")
+        self.player1_turn = True
+        self.game_over = False
+        self.turn_one = True
+
+    # def select_subgrid(
+    #     self, number: typing.Literal[0, 1, 2, 3, 4, 5, 6, 7, 8]
+    # ) -> np.array:
+    #     """Choose subgrid game to play."""
+    #     pass
+
+    def draw_board(self, term: blessed.Terminal) -> None:
```

Comment:
I agree this is super messy and needs refactoring. I want to change this to draw just the main board, then call redraw_subgrids to draw not just the dots but also the inner grid lines since we will need to redraw them later (to show active grid/grid winner). Working on this now.